desktop/extensions-sdk/build/set-up/backend-extension-tutorial.md

```diff
@@ -0,0 +1,419 @@
+{: .important}
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#go-app" data-group="go">Go</a></li>
+  <li><a data-toggle="tab" data-target="#node-app" data-group="node">Node</a></li>
+  <li><a data-toggle="tab" data-target="#python-app" data-group="python">Python</a></li>
+  <li><a data-toggle="tab" data-target="#java-app" data-group="java">Java</a></li>
+  <li><a data-toggle="tab" data-target="#net-app" data-group="net">.Net</a></li>
+</ul>
+
+<div class="tab-content">
+  <div id="go-app" class="tab-pane fade in active" markdown="1">
+
+```go
+package main
+
+import (
+	"flag"
+	"log"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/labstack/echo"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	var socketPath string
+	flag.StringVar(&socketPath, "socket", "/run/guest/volumes-service.sock", "Unix domain socket to listen on")
+	flag.Parse()
+
+	os.RemoveAll(socketPath)
+
+	logrus.New().Infof("Starting listening on %s\n", socketPath)
+	router := echo.New()
+	router.HideBanner = true
+
+	startURL := ""
+
+	ln, err := listen(socketPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	router.Listener = ln
+
+	router.GET("/hello", hello)
+
+	log.Fatal(router.Start(startURL))
+}
+
+func listen(path string) (net.Listener, error) {
+	return net.Listen("unix", path)
+}
+
+func hello(ctx echo.Context) error {
+	return ctx.JSON(http.StatusOK, HTTPMessageBody{Message: "hello world"})
+}
+
+type HTTPMessageBody struct {
+	Message string
+}
+```
+
+  </div>
+  <div id="node-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Node yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Node)
+> and let us know you'd like a sample for Node.
+{: .important }
+
+  </div>
+  <div id="python-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Python yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Python)
+> and let us know you'd like a sample for Python.
+{: .important }
+
+  </div>
+  <div id="java-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Java yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Java)
+> and let us know you'd like a sample for Java.
+{: .important }
+
+  </div>
+  <div id="net-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for .Net. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=.Net)
```

Comment:
could we take the example that Felipe put together for .NET ? (absolutely not a blocker here, we can merge this PR without this)

Reply:
I thought about it and actually, it contains a lot of things I don't master. So I was planning to add it in another PR with him to see what are the minimum pieces mandatory to put here

Reply:
👍 totally fine, keep this for later, and not in this PR